src/openrct2/drawing/NewDrawing.cpp

```diff
@@ -190,6 +190,16 @@ void gfx_fill_rect(rct_drawpixelinfo* dpi, int32_t left, int32_t top, int32_t ri
     }
```

Comment:
Make the old function call the new one, to avoid duplicating code.

Reply:
I haven't replaced all occurrences, it's only in libopenrct2 for now.

Reply:
I mean, make the old function just call the new `gfx_fill_rect` fn. They will both exist still, but the old will just route to the new :)